platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/AbstractPKIHandler.java

```diff
@@ -101,9 +101,20 @@ public HandlerResult getNormalizedToken(
       handlerResult.setToken(token);
       handlerResult.setStatus(HandlerResult.Status.COMPLETED);
     } else {
-      String errorMsg = "The certificate used to complete the request has been revoked.";
-      LOGGER.info(errorMsg);
-      throw new ServletException(errorMsg);
+
+      if (httpResponse == null) {
+        LOGGER.error(
+            "Error returning revoked certificate request because the HTTP response object is invalid.");
+      } else {
+
+        try {
+          httpResponse.sendError(HttpServletResponse.SC_FORBIDDEN, "Your certificate is revoked.");
+          httpResponse.flushBuffer();
```

Comment:
Do we no longer want to log something if the certificate has been revoked?

Reply:
I'll go ahead and add that back in